spec/beef/security_checks_spec.rb

```diff
@@ -1,17 +1,15 @@
 RSpec.describe 'BeEF Security Checks' do
-
   it 'dangerous eval usage' do
     Dir['**/*.rb'].each do |path|
       File.open(path) do |f|
         next if /#{File.basename(__FILE__)}/.match(path) # skip this file
-        next if /\/msf-test\//.match(path) # skip this file
-        next if /extensions\/dns/.match(path) # skip this file
+        next if %r{\/msf-test\/}.match(path) # skip this file
+        next if %r{extensions\/dns}.match(path) # skip this file
```

Comment:
One of the benefits of using `%r{}` format is that `/` no longer needs to be escaped as `\/`.

```suggestion
        next if %r{/msf-test/}.match(path) # skip this file
        next if %r{extensions/dns}.match(path) # skip this file
```

Except that these look like string matches, in which case there's no reason to use regex at all.

```suggestion
        next if path.include?('/msf-test/')
        next if path.include?('extensions/dns')
```